modules/govuk/manifests/apps/ckan/cronjobs.pp

```diff
@@ -30,6 +30,41 @@
     false => 'absent',
   }
 
+  govuk::apps::ckan::paster_cronjob { 'Find a Tender harvester run':
+    ensure         => $ensure,
+    paster_command => 'harvester run_test find-tender-data-harvest',
+    plugin         => 'ckanext-harvest',
+    hour           => '7',
```

Comment:
Sorry @brucebolt I forgot to update the card, the user has specified they'd like this harvester to run daily at 2.30am

Reply:
No problem, I've changed the time